### ISSUE TITLE
Print Beaker job URL in logs

### ISSUE THREAD

**Issue by spbnick** [RH-FMK/skt#75]
When a Beaker job is submitted, print its web UI URL in the logs. This way it's easier to open it in a browser and see its progress when looking through or watching skt logs.

**Comment by veruu** [comment]
`beaker-client` doesn't provide any debug / verbose output which logs hub URL. The only way how to get webUI URL of a job is parse hub URL out from the config file and concatenate it with `jobs/JOBID`

**Comment by spbnick** [comment]
Yes, it appears we'll have to do something like that.

**Comment by spbnick** [comment]
This is making it difficult to troubleshoot failures. Bumping up the priority.

**Comment by spbnick** [comment]
Right. We don't have priority labels. I would propose something like "pri:low", "pri:med", and "pri:high" labels. Green, yellow, and red, respectively. @veruu, @major?

**Comment by veruu** [comment]
Priority label work for me :+1:
